drivers/hmis/app/graphql/types/forms/form_definition.rb

```diff
@@ -107,8 +107,17 @@ def project
       object.filter_context&.fetch(:project, nil)
     end
 
+    # Context can optionally include an "active date", so that funder-based rules
+    # only consider funders that are active on the specified date.
+    def active_date
+      object.filter_context&.fetch(:active_date, nil) || Date.current
+    end
+
     def project_funders
-      project ? load_ar_association(project, :funders) : []
+      return [] unless project.present?
+
+      funders = load_ar_association(project, :funders)
+      funders.to_a.select { |f| f.active_on?(active_date) }
```

Comment:
@ttoomey question for you.. is it still preferable to use the loader here (as opposed to `project.funders.open_on_date(active_date`), even though we're only resolving 1 form definition at a time? I'm not sure how to figure that out.